src/components/admin/MainIssueController.jsx

```diff
@@ -156,8 +156,28 @@ export default class MainIssueController extends FalcorController {
               return false;
             }
           }
+          const urlR = /<a.*?href\s*?=\s*?["'](?!http)(.*?)["'] *?>/;
```

Comment:
Sorry, I don't like this variable name haha.

Can we please make it something like `absoluteUrlRegex`? `urlR` isn't very descriptive